pokemongo_bot/__init__.py

```diff
@@ -58,6 +59,15 @@ def update_web_location(self, cells=[], lat=None, lng=None, alt=None):
                 cell_id=cellid
             )
             response_dict = self.api.call()
+            for i in range(1,3):
```

Comment:
I think that if we wanted retry logic on the api, we would add it at the API layer instead of in this one instance where we call it.